### PR TITLE
Test a new release action

### PR DESCRIPTION
**Pull request by redeco**
...

### CODE REVIEW

**Comment by Nazarhanov** [comment]
Test reopen

File: .github/workflows/release-action.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+      - name: Release to GitHub
+        working-directory: ./.github/workflows
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GIT_AUTHOR_NAME: test-bot
+          GIT_AUTHOR_EMAIL: test@test.repo
+          GIT_COMMITTER_NAME: test-bot
+          GIT_COMMITTER_EMAIL: test@test.repo
+        run: npx semantic-release
```
Comment (Nazarhanov) [inline .github/workflows/release-action.yml:33]: Test review comment

File: .github/workflows/release-action.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: "Release to GitHub"
```
Comment (Nazarhanov) [inline .github/workflows/release-action.yml:10]: Test start a review

Comment (Nazarhanov) [reply .github/workflows/release-action.yml:10]: No quotes in short comments

File: .github/workflows/release-action.yml (hunk 1 above)
Comment (Nazarhanov) [reply .github/workflows/release-action.yml:33]: Additional comment

File: .github/workflows/release-action.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Add plugin for conventional commits
```
Comment (Nazarhanov) [inline .github/workflows/release-action.yml:21]: Additional comment

File: .github/workflows/release-action.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    name: "Release to GitHub"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
```
Comment (Nazarhanov) [inline .github/workflows/release-action.yml:19]: Use latest setup action and node version

File: .github/workflows/.releaserc [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+branches:
+  - master
+
+plugins:
+  - - "@semantic-release/commit-analyzer"
+    - preset: conventionalcommits
```
Comment (Nazarhanov) [inline .github/workflows/.releaserc:7]: Test review comment

File: .github/workflows/.releaserc [hunk 6]
```diff
@@ -0,0 +1,12 @@
+---
+branches:
+  - master
+
```
Comment (redeco) [inline .github/workflows/.releaserc:4]: Test comment

File: .github/workflows/.releaserc (hunk 5 above)
Comment (redeco) [reply .github/workflows/.releaserc:7]:
```suggestion
plugins:
  - - "@semantic-release/commit-analyzer"
    - preset: conventionalcommits
```

**Review by Nazarhanov — APPROVED** [review]

**Review by Nazarhanov — DISMISSED** [review]

**Review by Nazarhanov — COMMENTED** [review]
Changed review

**Pull request merged**